server/api/auth/loginWithIdp.js

```diff
@@ -129,6 +129,6 @@ module.exports = (err, user, req, res, clientID, idpId) => {
         }
       );
 
-      res.redirect(`${rootUrl}${defaultConfirm}#`);
+      res.redirect(`${rootUrl}/confirm#`);
```

Comment:
Why has this been reverted to be a hard-coded endpoint?

Reply:
Same thing here, this was just for testing but I didn't remove the WIP commit.